test/test_dynamic_shapes.py

```diff
@@ -12,12 +12,13 @@ def test_simple_expand(self):
     size2 = 2
     t1 = torch.zeros([size1, size2], device=dev)
     t1[3][0] = 1
+    t1[3][1] = 1
     # t2 has size [<=10, 2]
     t2 = torch.nonzero(t1)
     t5 = torch.ones(1, device=dev)
-    t5.expand(t2.size(0))
-    t5_cpu = t5.cpu()
-    self.assertEqual(t5_cpu.shape[0], 1)
+    t6 = t5.expand(t2.size(0))
```

Comment:
@JackCaoG you made a comment in https://github.com/pytorch/xla/pull/4212/files "I felt like we should add a check here to test t5.shape(), this can be down in a follow up pr.".

Regarding the comment, I think you want to check t6.shape() according to this PR. Can you explain why you want to check t4.shape() and how is it different from what we are doing right now `self.assertEqual(t6_cpu.shape[0], 2)`?

Reply:
yea, t6 is find, I thought expand is a inplace operation.

Reply:
can you also add a check for `t6`'s ir, by using `print(torch_xla._XLAC._get_xla_tensors_text([t6]))`, I think you should see something like `<=10` in the IR.

Reply:
Yeah, I added the print after `self.assertEqual(t6_cpu.shape[0], 2)` and it prints 
```
IR {
  %0 = f32[<=10]{0} xla::device_data(), device=TPU:0, ROOT=0
}
```
Is there any you want to check here?

Reply:
Or do you want to check `<=10` is in `torch_xla._XLAC._get_xla_tensors_text([t6])`?

Reply:
```
Or do you want to check <=10 is in torch_xla._XLAC._get_xla_tensors_text([t6])?
```
yea, I want to make sure that `<=10` works

Reply:
done